multimap/src/test/java/org/infinispan/multimap/impl/MultimapSCI.java

```diff
@@ -3,10 +3,11 @@
 import org.infinispan.protostream.SerializationContextInitializer;
 import org.infinispan.protostream.annotations.AutoProtoSchemaBuilder;
 import org.infinispan.test.TestDataSCI;
+import org.infinispan.test.data.Person;
 
 @AutoProtoSchemaBuilder(
       dependsOn = TestDataSCI.class,
-      includeClasses = SuperPerson.class,
+      includeClasses = {Person.class, SuperPerson.class},
```

Comment:
Nitpick, could you split the array over multiple lines like in the other` @AutoProtoSchemaBuilder`